lib/redis_client/sentinel_config.rb

```diff
@@ -159,5 +165,19 @@ def each_sentinel
 
       raise last_error if last_error
     end
+
+    def refresh_sentinels(sentinel_client)
+      sentinel_response = sentinel_client.call("SENTINEL", "sentinels", @name, &@to_list_of_hash)
+      sentinels = sentinel_response.map do |sentinel|
+        { host: sentinel.fetch("ip"), port: sentinel.fetch("port").to_i }
```

Comment:
```suggestion
        { host: sentinel.fetch("ip"), port: Integer(sentinel.fetch("port")) }
```

For consistency and strictness. Shouldn't do any difference in practice.